tests/app/questionnaire/test_placeholder_transforms.py

```diff
@@ -118,6 +118,23 @@ def test_concatenate_list(self):
             == "Milk, Eggs, Flour, Water"
         )
 
+    def test_add(self):
+        assert self.transforms.add(1, 2) == 3
+        assert self.transforms.add("1", 2) == 3
+
+    def test_format_ordinal(self):
+        assert self.transforms.format_ordinal(1, "a_or_an") == "a 1st"
```

Comment:
As above, maybe split into 2 prefix, suffix

Reply:
I moved it to "prefix" and "suffix" tests now, thanks👍